### PR TITLE
feature/add-logo

### PR DESCRIPTION
**Pull request by hermsicle**

### CODE REVIEW

File: herman.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Back End Development is hella cool</h1>
```
Comment (awwmicky) [inline herman.html:10]: what happen, you ain't full stack

**Review by awwmicky — CHANGES_REQUESTED** [review]
you go fix this thing

File: herman.html (hunk 1 above)
Comment (hermsicle) [reply herman.html:10]: What should I fix?

**Review by awwmicky — APPROVED** [review]
yes Shang-chi san

**Pull request merged**